### PR TITLE
DOC add logo

### LINKED ISSUE #2: LOGO
Would be nice to have a logo for the project.

### PR DESCRIPTION
**Pull request by adrinjalali**
Fixes #2 

Add logo to sphinx and readme.

### CODE REVIEW

File: README.rst
Comment (adrinjalali) [inline README.rst:26]: this will only be rendered correctly once merged.

**Review by BenjaminBossan — APPROVED** [review]

**Comment by merveenoyan** [comment]
LOGO
can we post this instead? @skops-dev/maintainers

**Pull request merged**
